packages/foreman/rubygem-journald-logger/rubygem-journald-logger.spec

```diff
@@ -76,10 +76,14 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_docdir}
 %{gem_instdir}/Gemfile
 %doc %{gem_instdir}/README.md
+%doc %{gem_instdir}/CHANGELOG.md
 %{gem_instdir}/Rakefile
 
 %changelog
-* Wed Sep 05 2018 Eric D. Helms <ericdhelms@gmail.com> - 2.0.3-2
+* Thu Oct 11 2018 Lukas Zapletal <lzap+rpm@redhat.com> 2.0.4-1
+- Bumped version for Ruby 2.0 support
+
+* Wed Sep 05 2018 Eric D. Helms <ericdhelms@gmail.com> 2.0.3-2
```

Comment:
Nit: I'd appreciate keeping the `- ` in Erics changelog to have a minimal change.